bin/rsync-fullexport-files

```diff
@@ -15,6 +15,7 @@ if [ -n "$RSYNC_FULLEXPORT_HOST" ]; then
         --archive \
         --delete \
         --exclude='latest-is*' \
+        --exclude='mbscript-*' \
```

Comment:
Is this related to this patch?

Reply:
Yes; See the message of commit <https://github.com/metabrainz/musicbrainz-server/pull/2350/commits/5d3a2af1e87d089aaaf94019d0b118625fdaa194>:

> To prevent publishing temporary files, those are excluded from 'rsync'.

Indeed the name of the created temporary directories start iwth `mbscript-`; See https://github.com/metabrainz/musicbrainz-server/blob/v-2021-11-29/admin/functions.sh#L9